src/rpc.rs

```diff
@@ -278,6 +278,10 @@ impl State {
         self.pubsub.subscription_active(key)
     }
 
+    fn can_subscribe(&self) -> bool {
```

Comment:
should be `caching_allowed` or something like that because it disables not just subscriptions but also saving values into cache

Reply:
Changed the method name to `is_caching_allowed`, but only here, as it makes sense for pubsub